website/docs/r/pki_secret_backend_cert.html.md

```diff
@@ -84,3 +84,5 @@ In addition to the fields above, the following attributes are exported:
 * `serial_number` - The serial number
 
 * `expiration` - The expiration date of the certificate in unix epoch format
+
```

Comment:
nit: mind folding line longer than 100 columns?

Reply:
I am happy to do this if this is the current prevailing style. I was following the example of `min_seconds_remaining` and `auto_renew` above, which do not have wrapping... but now I look more broadly I see that `namespace` _does_ have some (incomplete) wrapping at 80 columns.

Can you confirm which of these is the current style that new documentation should follow? (and should I update the existing counterexamples to match while I'm here, or would you prefer that sort of change to arrive in a separate PR?)

Reply:
The style is highly inconsistent unfortunately. We can punt this to future work. No need to address it now.